utils/database.js

```diff
@@ -108,10 +109,21 @@ const queryDatabase = function(database, sql, values) {
     });
 }
 
-const tableFields = function(table) {
-    const sql = "SELECT column_name FROM information_schema.columns WHERE table_name = ?"
-    return poolPromise.then((p) => {
-        return p.query(sql, [table]);
+const tableFields = function(table, database) {
+    const sql = "SELECT column_name FROM information_schema.columns WHERE table_name = ? AND table_schema = ?"
```

Comment:
did we ever find out why it needed this list of columns at runtime?

Reply:
It's not 100% clear where the logic goes, I logged the queries and hit a few endpoints and didn't see a whole lot, but it would take more to confidently replace the queries with their static equivalents. Might also relate to e.g. query parameters that are not specified in most cases.